README.md

```diff
@@ -1,2 +1,50 @@
-# crypto-hunter
-This is a mobile based web application built using react.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+## Crypto Hunter
+This is a mobile based web application that provides numercial data of any crypto currency.
+It consists of three pages: 
+   - Home Page: this pages displays a list of crypto currencies together with their name, symbol and current price in the market.
+   - Details Page: this page displays numerical information about the chosen coin.
+   - HistoricalChart Page: this page displays the line and bar graph represention of the coins numerical data.
+ 
+### Built With
+- React
+- Redux
+- CSS
+- JSX
+- JavaScript
+- Chart.js
+- API
+  
+### Live Demo
+[Crypto Hunter](https://admirable-chimera-5ddbb5.netlify.app/)
+[Vedio](https://drive.google.com/file/d/1xoXsbl5HFcQWJ6SYrD5sAt_o1wvtSeIz/view?usp=sharing)
```

Comment:
- [ ] Your video link and the live link appear under the live demo heading section. For much better documentation, and to avoid confusion I would recommend that you put the video in its own section.

  You can use this markdown syntax
  ```
  ### Live Demo
  - [Crypto Hunter](https://admirable-chimera-5ddbb5.netlify.app/)
  
  ### Video
  - [Video](https://drive.google.com/file/d/1xoXsbl5HFcQWJ6SYrD5sAt_o1wvtSeIz/view?usp=sharing)
  ```

  ![dee](https://user-images.githubusercontent.com/54848363/189133125-7373b80d-0de3-4651-895f-51d68467cf22.jpg)

Reply:
Thank you for your feedback. I will correct it.